be/test/exec/vectorized/parquet_scanner_test.cpp

```diff
@@ -365,6 +416,72 @@ TEST_F(ParquetScannerTest, test_parquet_data_with_3_column_from_path) {
     test_column_from_path<false>(columns_from_file, columns_from_path, column_values, dst_slot_exprs);
 }
 
+TEST_F(ParquetScannerTest, test_to_json) {
```

Comment:
PARALLEL_TEST？

Reply:
I'm afraid not. `TEST_F` is a fixture test, which is bind to test fixture `ParquetScannerTest`.